proto/info/edge_node_cluster.proto

```diff
@@ -0,0 +1,237 @@
+// Copyright(c) 2024 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.info;
+option go_package = "github.com/lf-edge/eve-api/go/info";
+option java_package = "org.lfedge.eve.info";
+
+import "google/protobuf/timestamp.proto";
+import "info/info.proto";
+
+enum KubeNodeStatus {
```

Comment:
I assume this and other enums come from some kubernetes/longhorn source. Would it make sense to put comments indicating the source? That might be helpful if additional values are added later.

Reply:
Yes, I can add them.